config/config.go

```diff
@@ -1,34 +1,48 @@
 package config
 
 import (
+	"fmt"
 	"io/ioutil"
-	"github.com/hashicorp/hcl"
-	"github.com/gruntwork-io/terragrunt/dynamodb"
-	"github.com/gruntwork-io/terragrunt/remote"
+
 	"github.com/gruntwork-io/terragrunt/errors"
+	"github.com/gruntwork-io/terragrunt/locks"
+	"github.com/gruntwork-io/terragrunt/remote"
+	"github.com/hashicorp/hcl"
 )
 
-const TERRAGRUNT_CONFIG_FILE = ".terragrunt"
+const configFilePath = ".terragrunt"
+
+// Config represents a parsed and expanded configuration
+type Config struct {
```

Comment:
Here too I prefer the longer name. There are undoubtedly thousands of types called `Config` in the Go world. Seeing the type elsewhere in the code would tell me nothing about its intent without having to dig into its definition and documentation. The package name (also `config`) isn't much help here either. That's why I used `TerragruntConfig` originally.